src/main/java/frc/robot/subsystems/IntakeLimelight.java

```diff
@@ -364,8 +365,81 @@ public Optional<Double> getXOfSmallestY() {
     return Optional.of(corners[minYIdx - 1]);
   }
 
+  public Optional<Double> getObjectHeightPx(double[] corners) {
```

Comment:
Could someone explain what the corners array is and what are they used for?

Reply:
Basel knows more about vision than I do, but in order to get the object we're getting an array of "corners" of the cone from the limelight and analyzing those.

Reply:
Literally just the pixel values of all the corners of the cone shape in the camera image, we get the cone in the image and that let's us calculate where the cone is in real life relative to the robot based on this: https://docs.limelightvision.io/en/latest/theory.html#from-pixels-to-angles